contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -106,3 +120,21 @@ func SpawnCheck() {
 		}
 	}()
 }
+
+func GetBlockGasLimit(header *types.Header, state vm.StateDB) (uint64, error) {
+	var gasLimit *big.Int
+	_, err := contract_comm.MakeStaticCall(
+		params.BlockchainParametersRegistryId,
+		blockchainParametersABI,
+		"blockGasLimit",
+		[]interface{}{},
+		&gasLimit,
+		params.MaxGasForBlockGasLimit,
```

Comment:
Should we have a shared `MaxGasForReadingBlockchainParameter` parameter? Since presumably the cost will be the same for all?